go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook
 
-go 1.20
+go 1.21
```

Comment:
I don't think we can update this until we drop 1.20 support.

Reply:
I think an earlier upgrade is better for us, or sometime soon some cve will be open to fix the golang version, do we need to wait for something before upgrading?